### DISCUSSION TITLE
Briefkasten V2 - Looking for Feedback

### DISCUSSION THREAD

**Discussion by ndom91** [ndom91/briefkasten#65] [Announcements]
Hey folks! :wave: 

First of all, I'd like to thank everyone who gave my "side project" a shot and especially those that continue to use it today!

As I've recently had some more time on my hands, I decided to take another stab at the web app portion of this application and am rewriting it using SvelteKit and Typescript. You can check out the progress at [ndom91/sveltekasten](https://github.com/ndom91/sveltekasten). I plan on bringing the code over in a new branch to this repository eventually though. My initial thoughts are that the existing Next.js 12 codebase will continue to live on in a `v1` branch and this new one will live on as `main`, but nothing is set in stone yet.. Anyway, here are some initial prototype screenshots:

<table>
<tr>
<td>

![image](https://github.com/ndom91/briefkasten/assets/7415984/8c85141a-fc15-4d47-994e-a93610f46196)

</td>
<td>

![image](https://github.com/ndom91/briefkasten/assets/7415984/969fe900-6405-4316-aceb-e2f2a16aeee2)

</td>
</tr>
</table>

The keen eyed among you might have already shotted one major new feature - RSS Feeds!

So currently my focus is on reimplementing all previous major features and improving / adding:
- RSS feed support
- Much easier self-hosting (including screenshotting!)
- Keyboard navigation
- Bette mobile support
- Squash all smaller bugs

And that brings me to the real point of this message today - what else would you all like to see?! Beside for scratching my own itches, I'd love to get some more feedback from users and incorporate what I can :heart: 

Thanks again,
Nico

**Comment by ndom91** [comment]
Anyone following along here, I've released a beta version you can play with [here](https://dev.briefkastenhq.com/settings) :partying_face: 

None of the existing logins / bookmarks are there, as **the connected database will be wiped after the beta period**.

But **I do plan on migrating the existing data from v1 to v2** when the beta period is over and I deprecate v1 for this version.

**Comment by kabatp** [comment]
How is the development going? Is there any ETA for the v2?
